src/core_read.cpp

```diff
@@ -45,6 +45,17 @@ CScript ParseScript(const std::string& s)
         }
     }
 
+    auto it = mapOpNames.find(s);
+    if (it == mapOpNames.end()) throw std::runtime_error("script parse error");
```

Comment:
You could be more specific about the error here. E.g. like the out of range error:
```c++
throw std::runtime_error("script parse error: unknown opcode");
```

Reply:
Done.